packages/xstate-vue/package.json

```diff
@@ -41,7 +41,7 @@
     "url": "https://github.com/davidkpiano/xstate/issues"
   },
   "peerDependencies": {
-    "@vue/composition-api": "^0.3.4",
+    "@vue/composition-api": "^0.6.5",
```

Comment:
do we depend on some features in `0.6.5` specifically or could we change this to `^0.6.0`?

Reply:
we can change it. there were bugs caught in 0.6.0->.4 within a few days so didn't want to point to versions that were knowingly buggy but I suppose that's for the consumer to figure out? Was curious why it wasn't ^0.3.0